Lib/gzip.py

```diff
@@ -17,6 +17,14 @@
 
 READ, WRITE = 1, 2
 
+# OS field values as specified in RFC 1952
+OS_CODES = {
+    'linux': 3,
+    'os2': 6,
+    'win32': 11,
+    'unknown': 255
```

Comment:
Is there any reason not to use bytes value?
e.g: 'unknown': b'\xff'

Reply:
No reason, really. Fixed in a4a0e6866b8644fd8ea3b3aea405cf8e40daa1ad

Reply:
Fixed a4a0e6866b8644fd8ea3b3aea405cf8e40daa1ad